### DISCUSSION TITLE
Incorrect listing of Irish counties

### DISCUSSION THREAD

**Discussion by s-flux** [dr5hn/countries-states-cities-database#728] [General]
Hi you have the incorrect listing of Irish counties (stats) in your database.
You have included the four Provinces they are not postal address and you are missing the county of Leitrim.

Here is the correct list for the Irish Republic = 26 counties (states)

Carlow
Cavan
Clare
Cork
Donegal
Dublin
Galway
Kerry
Kildare
Kilkenny
Laois
Leitrim
Limerick
Longford
Louth
Mayo
Meath
Monaghan
Offaly
Roscommon
Sligo
Tipperary
Waterford
Westmeath
Wexford
Wicklow

**Comment by ejohanson-vi3** [comment]
The database lists Connacht, Leinster, Munster, and Ulster as provinces, and not counties.
It is true that Leitrim county is missing though.
